sdk/instrumentation/net/http/handler.go

```diff
@@ -103,25 +93,14 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 				isMultipartFormDataBody)
 		}
 
-		processingBody := body
-		if int(h.dataCaptureConfig.BodyMaxProcessingSizeBytes.Value) < len(body) {
-			processingBody = body[:h.dataCaptureConfig.BodyMaxProcessingSizeBytes.Value]
```

Comment:
`dataCaptureConfig.BodyMaxProcessingSizeBytes` config is also becoming obsolete as we are using the body stored as span attribute for filter evaluation

Reply:
Yes, I'm ok with that.

Reply:
Can you help open a ticket to deprecate body max processing size throughout? Thanks!

Reply:
yes, :https://traceableai.atlassian.net/browse/ENG-37604